### PR TITLE
PR for Calculator route

### PR DESCRIPTION
**Pull request by Miliyonayalew**
## In this milestone I have
- created a website consisting of 3 pages: `Home`, `Calculator`, and `Quote`
- Designed layout using the wireframe
- Add my own style to improve the look and feel
- Created a `route `for every page
- Used React-Router (`BrowserRouter `as `Router `and `Routes`, `Route`, and `NavLink`).

### CODE REVIEW

File: src/components/calculator.css [hunk 1]
```diff
@@ -77,3 +77,97 @@ button {
 button:hover {
   cursor: pointer;
 }
+
+.navbar__logo {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
```
Comment (BenMukebo) [inline src/components/calculator.css]: - [ ] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code to No repeating of the same rule by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```css
.d-flex {
  display: flex;
}

.flex-center {
  justify-content: center;
  align-items: center;
}
```
- JS file
```js
<div className="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Miliyonayalew,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- Gitflow was correctly used.  ✔️   
- There is a dedicated route for every page. ✔️ 
- Navigation links for all three routes is displayed on each page. ✔️ 
- You have followed  JavaScript best practices. ✔️ 
- The layout match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md). 🥇 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Great work with your readme but I would like to recommend removing some unnecessary information from your readme and keeping only the main header section. For example, you can summarize how to run the project locally. Please fix it in order to make your readme more professional :+1: 
Screenshot from 2022-11-01 20-43-27

 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.css (hunk 1 above)
Comment (Miliyonayalew) [reply src/components/calculator.css]: Thank you @BenMukebo  for your feedbacks, Happy Coding 😄

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @Miliyonayalew,

Good job on making the changes asked by the previous reviewer.

Your project is complete!  

## Highlights:
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ Good descriptive PR 👍 
✅ Good commit messages 👍 
✅ Your app is nicely styled and all features work fine as expected 👍 

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Miliyonayalew** [comment]
Thank you @AmaduKamara for the review, Happy coding 😄

**Pull request merged**